components/Partners/Partners.js

```diff
@@ -0,0 +1,15 @@
+import Section from '../Section';
+import styles from './Partners.scss';
+
+const Partners = () => (
+  <Section className={styles.partners}>
```

Comment:
Same as the teams.json feedback: Load the Partner items from an independent json file?